api/app/serializers/spree/api/v2/platform/taxon_serializer.rb

```diff
@@ -0,0 +1,38 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class TaxonSerializer < BaseSerializer
```

Comment:
I could not use `include ::Spree::Api::V2::ResourceSerializerConcern` here, is this because a taxon is self nesting?